addon/services/apollo.js

```diff
@@ -26,7 +27,7 @@ function newDataFunc(observable, resultKey, resolve) {
   let mergedProps = { _apolloObservable: observable };
 
   return ({ data }) => {
-    let dataToSend = isNone(resultKey) ? data : data[resultKey];
+    let dataToSend = isNone(resultKey) ? data : get(data, resultKey);
```

Comment:
I was thinking about to send a PR with this change. With `get`, I guess I can get a nested property

```javascript
return this.get('apollo').query({ query }, 'pretty.nested.property');
```